src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java

```diff
@@ -29,7 +29,7 @@
 
 
 /**
- * SQLServerStatment provides the basic implementation of JDBC statement functionality. It also provides a number of
+ * Provides the basic implementation of JDBC statement functionality. It also provides a number of
```

Comment:
Provides an implementation of <code>java.sql.Statement</code> JDBC Interface to assist in creating Statements against SQL Server.

(might be able to word better but basic idea is to specify implemented interface, and the basic functionality of class against SQL Server)